src/schema/v2/me/userInterests.ts

```diff
@@ -34,7 +34,7 @@ export const userInterestCategoryEnum = new GraphQLEnumType({
 
 export const userInterestInterestUnion = new GraphQLUnionType({
   name: "UserInterestInterest",
-  types: [ArtistType, GeneType],
+  types: () => [ArtistType, GeneType],
```

Comment:
this change is to resolve `ArtistType` being `undefined`, see [this slack thread 🔒](https://artsy.slack.com/archives/CP9P4KR35/p1643790512633099) for more context